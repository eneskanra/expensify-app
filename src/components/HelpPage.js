import React from "react";

const HelpPage = () => (
    <div>This from my help component</div>
);

export default HelpPage;