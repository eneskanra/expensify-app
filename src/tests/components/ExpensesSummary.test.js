import React from "react";
import { shallow } from "enzyme";
import { ExpensesSummary } from "../../components/ExpensesSummary";

test('should correctly render ExpensesSummary with 1 expense', () => {
    const wrapper = shallow(
        <ExpensesSummary
        expensesCount={1}
        expensesTotal={235}></ExpensesSummary>
    );
    expect(wrapper).toMatchSnapshot();
});

test('should correctly render ExpensesSummary with multiple expense', () => {
    const wrapper = shallow(
        <ExpensesSummary
        expensesCount={22}
        expensesTotal={223455555}></ExpensesSummary>
    );
    expect(wrapper).toMatchSnapshot();
});