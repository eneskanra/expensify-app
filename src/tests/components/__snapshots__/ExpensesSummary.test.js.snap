// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render ExpensesSummary with 1 expense 1`] = `
<div>
  <h1>
    Viewing 
    1
     
    expense
     totalling 
    $2.35
  </h1>
</div>
`;

exports[`should correctly render ExpensesSummary with multiple expense 1`] = `
<div>
  <h1>
    Viewing 
    22
     
    expenses
     totalling 
    $2,234,555.55
  </h1>
</div>
`;
